### PR TITLE
MAUI-5183: Prepare Popup size UG for .NET MAUI Popup.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Prepared the Popup Size UG for .NET MAUI Popup.

### CODE REVIEW

File: MAUI/Popup/popup-size.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+            Button isOpenButton = new Button();
+            isOpenButton.Clicked += heightWidth_Clicked;
+            isOpenButton.Text = "Click to show popup";
+            stackLayout.Children.Add(isOpenButton);
+            this.Content = popup;
+        }
+
+        private void heightWidth_Clicked(object sender, EventArgs e)
+        {
+			popup.Show();
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-size.md:159]: check code alignment

File: MAUI/Popup/popup-size.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![Popup with HeightRequest and WidthRequest](Images/popup-size//maui-popup-with-heightrequest-and-widthrequest.png)
+
+## Full Screen
+
+SfPopup can be shown in full width and height of the screen using,
+
+  * `SfPopup.IsFullScreen`
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-size.md:173]: `IsFullScreen` property
`Show(bool isFullScreen)`

File: MAUI/Popup/popup-size.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+        <StackLayout Padding="20">
+            <Button x:Name="heightWidth" Text="HeightWidth" Clicked="heightWidth_Clicked" WidthRequest="300"/>
+            <popup:SfPopup x:Name="popup" HeaderTitle="ListView" HeightRequest="320" ShowCloseButton="True" ShowFooter="True">
+                <popup:SfPopup.ContentTemplate>
+                    <DataTemplate>
+                        <listView:SfListView  x:Name="listView"  ScrollBarVisibility="Never" ItemSpacing="5" WidthRequest="320" 
+                          ItemsSource="{Binding contactsinfo}" 
+                          >
+                            <listView:SfListView.ItemTemplate>
+                                <DataTemplate>
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-size.md:38]: No need to show the full customization.
Highlight the Property setting (WidthRequest, HeightRequest

File: MAUI/Popup/popup-size.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+        ContactsViewModel viewModel;
+        SfPopup popup;
+
+        public MainPage()
+        {
+            InitializeComponent();
+            listView = new SfListView() { ItemSpacing = 5 };
+            listView.WidthRequest = 350;
+            listView.ScrollBarVisibility = ScrollBarVisibility.Never;
+            listView.ItemTemplate = new DataTemplate(() =>
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-size.md:102]: No Need to show full codes.
Highlight required lines.

File: MAUI/Popup/popup-size.md [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+
+    <?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PopupMaui.AutoSizePage"
+              xmlns:popup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup">
+    <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="show" Text="Show" WidthRequest="300" Clicked="show_Clicked"/>
+            <popup:SfPopup x:Name="popup" AutoSizeMode="Height">
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-size.md:306]: Move AutoSizeMode line to next line and highlight it.

File: MAUI/Popup/popup-size.md [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+        {
+			popup.Show();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Popup with HeightRequest and WidthRequest](Images/popup-size//maui-popup-with-heightrequest-and-widthrequest.png)
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-size.md:167]: .NET MAUI Popup

change as .NET MAUI Popup wherever used Popup (in all image alt text)

File: MAUI/Popup/popup-size.md [hunk 7]
```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Popup Size in MAUI Popup control | Syncfusion
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md:3]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md (hunk 4 above)
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md:73]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md (hunk 1 above)
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md (hunk 2 above)
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md [hunk 8]
```diff
@@ -0,0 +1,333 @@
...
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PopupMaui.FullScreenPage"
+             xmlns:local="clr-namespace:PopupMaui"
+             xmlns:inputLayout="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
+             xmlns:popup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             >
+    <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="show" Text="Show Popup" Clicked="show_Clicked"/>
+            <popup:SfPopup x:Name="popup" AppearanceMode="TwoButton" AcceptButtonText="SAVE"  DeclineButtonText="CANCEL" HeaderHeight="60" FooterHeight="70" ShowCloseButton="True" ShowFooter="True">
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md (hunk 3 above)
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+{% endhighlight %}
+
+Executing the above codes renders the following output in windows.
+
+![.NET MAUI Popup with Full Screen](Images/popup-size//maui-popup-with-fullscreen.png)
+
+
+## Auto-size Popup
+
+The `SfPopup` can auto-size the popup view based on the contents loaded inside its `SfPopup.ContentTemplate` property using the `SfPopup.AutoSizeMode` property. The default value is `AutoSizeMode.None`. You can choose to auto-size the Popup in height, width or in both height and width of its contents. By default, the `HeightRequest` and `WidthRequest` set to the `SfPopup` or the views loaded inside the template is given higher priority than the `AutoSizeMode`.
```
Comment (M-A32) [inline MAUI/Popup/popup-size.md:178]: The `SfPopup` can auto-size the popup view based on the contents loaded inside its `SfPopup.ContentTemplate` property using the `SfPopup.AutoSizeMode` property. The default value is `AutoSizeMode.None`. You can choose to auto-size the Popup in the height, width or in both .height and width of its contents. By default, the `HeightRequest` and `WidthRequest` set to the `SfPopup` or the views loaded inside the template are given higher priority than the `AutoSizeMode.`

File: MAUI/Popup/popup-size.md [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+
+In the following code snippet the Popup is auto-sized in height based on the content loaded inside the `SfPopup.ContentTemplate` property.
```
Comment (M-A32) [inline MAUI/Popup/popup-size.md:180]: In the following code sample, the Popup is auto-sized in the height based on the content loaded inside the `SfPopup.ContentTemplate` property.

File: MAUI/Popup/popup-size.md [hunk 11]
```diff
@@ -0,0 +1,223 @@
...
+    <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="show" Text="Show" WidthRequest="300" ShowFooter="True" Clicked="show_Clicked"/>
+            <popup:SfPopup x:Name="popup" 
+            AutoSizeMode="Height"
+            >
+                <popup:SfPopup.ContentTemplate>
+                    <DataTemplate>
+                        <StackLayout Padding="5,5,5,0">
+                            <Label Text="Window loads under the parent window surrounded by an overlay which prevents clicking anywhere else on the screen apart from the control of the modal. Modal opens in the same window. It also does not require any user action to open, and give callbacks when closing or opening the modal."
```
Comment (M-A32) [inline MAUI/Popup/popup-size.md:200]: <Label Text=" Window loads under the parent window surrounded by an overlay which prevents clicking anywhere else on the screen apart from the control of the modal. Modal opens in the same window. It also does not require any user action to open, and give callbacks when closing or opening the modal."

**Review by M-A32 — CHANGES_REQUESTED** [review]

File: MAUI/Popup/popup-size.md [hunk 12]
```diff
@@ -0,0 +1,223 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with HeightRequest and WidthRequest](Images/popup-size//maui-popup-with-heightrequest-and-widthrequest.png)
+
+## Full Screen
+
+SfPopup can be shown in full width and height of the screen using,
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md [hunk 13]
```diff
@@ -0,0 +1,223 @@
...
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+
+        private void show_Clicked(object sender, EventArgs e)
+        {
+            // Shows SfPopup in full width and height of the screen using the SfPopup.IsFullScreen property.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md (hunk 9 above)
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

File: MAUI/Popup/popup-size.md (hunk 10 above)
Comment (RasikaPalanisamy) [reply MAUI/Popup/popup-size.md]: Addressed the concerns provided.

**Review by PiruthivirajM — APPROVED** [review]
approved

**Pull request merged**